contents/startups.mdx

```diff
@@ -120,6 +114,31 @@ import refer from './images/startup-refer.png'
   </div>
 </section>
 
+<section class="my-24 px-5">
+  <div class="max-w-screen-2xl mx-auto">
+    <h3 class="text-2xl lg:text-3xl m-0 text-center mb-6 sm:mb-16 lg:mb-8">Trusted by the world's most successful startups and accelerators</h3>
+    <div class="flex justify-center">
+      <ul class="m-0 p-0 list-none inline-grid sm:grid-cols-3 lg:grid-cols-3 justify-evenly relative text-center sm:text-left">
+        <li class="relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12 lg:px-8">
+          <GatsbyImage image={getImage(props?.images[10])} alt="YC logo" objectFit="contain"  align="center" style={{ maxWidth: 200 }} />
+          <h5 class="text-xl font-extrabold m-0 pb-1 pr-4">Y Combinator</h5>
+          <p class="m-0 text-[15px]">The world's leading startup accelerator relies on PostHog. A/B experiments have helped improve engagement by +40%, while gathering +30% more data than other tools. <a href="/customers/ycombinator">Read more.</a></p>
+        </li>
+        <li class="relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12 lg:px-8">
+          <GatsbyImage image={getImage(props?.images[11])} alt="AssemblyAI" objectFit="contain" style={{ maxWidth: 200 }} />
+          <h5 class="text-xl font-extrabold m-0 pb-1 pr-4">AssemblyAI</h5>
+          <p class="m-0 text-[15px]">AssemblyAI tried Mixpanel, but had to limit ingestion to avoid exorbitant bills. They switched to PostHog and now track much more data, without any throttling. <a href="/customers/assemblyai">Read more.</a></p>
+        </li>
+        <li class="relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12 lg:px-8">
+          <GatsbyImage image={getImage(props?.images[12])} alt="Goliath" objectFit="contain" style={{ maxWidth: 200 }} />
+          <h5 class="text-xl font-extrabold m-0 pb-1 pr-4">Goliath</h5>
+          <p class="m-0 text-[15px]">Goliath has put PostHog at the heart of it's product development cycle since joining the startup program. Dashboards and insights help them build better products, faster. <a href="blog/startup-goliath">Read more.</a></p>
```

Comment:
No, it didn't have one. I made the call to stop getting Lottie to do them, as they were so minimal value vs giving her time to focus on other things.

Reply:
```suggestion
          <p class="m-0 text-[15px]">Golioth has put PostHog at the heart of it's product development cycle since joining the startup program. Dashboards and insights help them build better products, faster. <a href="blog/startup-golioth">Read more.</a></p>
```

Reply:
Good spot on the URL. Corrected the name too. No idea why they spell it that way.